test/integ/test_job_submissions.py

```diff
@@ -59,16 +54,15 @@ def test_success(
 
     def test_job_run_as_user(
         self,
-        deadline_client: DeadlineClient,
-        farm: Farm,
-        queue_with_job_run_as_user: Queue,
-        job_run_as_user: PosixSessionUser,
+        deadline_client,
+        deadline_resources,
+        job_run_as_user,
     ) -> None:
         # WHEN
         job = Job.submit(
             client=deadline_client,
-            farm=farm,
-            queue=queue_with_job_run_as_user,
+            farm=deadline_resources.farm,
+            queue=deadline_resources.scaling_queue,
```

Comment:
Should we be using the `scaling_queue` here? Do we have the logic that sets up the scaling fleet?

Reply:
It is simply an autoscaling fleet with the autoscaling mode enabled at the moment. Should this be switched to one with no scaling?